core/src/main/java/org/apache/carbondata/core/dictionary/server/DictionaryServer.java

```diff
@@ -83,9 +81,12 @@ public ChannelPipeline getPipeline() throws Exception {
    */
   public void shutdown() throws Exception {
     DictionaryKey key = new DictionaryKey();
-    key.setType("WRITE_DICTIONARY");
+    key.setType(DictionaryKeyType.WRITE_DICTIONARY);
     dictionaryServerHandler.processMessage(key);
-    bootstrap.releaseExternalResources();
-    bootstrap.shutdown();
+    worker.shutdownGracefully();
```

Comment:
Method `bootstrap.group()` is deprecated, so can we use `EventLoopGroup` to shutdown?